extensions/wikia/Wall/Wall.setup.php

```diff
@@ -224,15 +224,21 @@
 $wgGroupPermissions['vstf']['wallarchive'] = true;
 $wgGroupPermissions['helper']['wallarchive'] = true;
 $wgGroupPermissions['sysop']['wallarchive'] = true;
+$wgGroupPermissions['moderator']['wallarchive'] = true;
 
 $wgGroupPermissions['*']['wallremove'] = false;
-$wgGroupPermissions['user']['wallremove'] = true;
+$wgGroupPermissions['user']['wallremove'] = false;
```

Comment:
Do we need to include this line here? I believe the user should be set to false by the wildcard on line 229.